.github/workflows/test.yml

```diff
@@ -4,14 +4,14 @@ on: [pull_request]
 
 jobs:
   build:
-
     env:
-      RELEASE: release-py37.yml
+      RELEASE: release-py3x.yml
+
 
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.6", "3.8"]
+        python-version: ['3.8', '3.10']
```

Comment:
How do you think it will impact the users? Have we communicated it beforehand?

Reply:
We are no longer offering updated Python 3.6 komodo builds, the last available one is 2022.06.